common/lib/sandbox-packages/verifiers/tests_draganddrop.py

```diff
@@ -787,7 +787,7 @@ def test_1(self):
         user_groups = [['1'], ['name_with_icon']]
         user_positions = [{'user': [[10, 10]]}, {'user': [[20, 20]]}]
 
-        self.assertEqual(correct_groups, dnd.correct_groups)
+        self.assertEqual(sorted(correct_groups), sorted(dnd.correct_groups))
```

Comment:
Is this still needed after https://github.com/edx/edx-platform/pull/21830 ?  The test seems to be passing on master now.